### PR TITLE
feat: validate wearable representations correspond to content files

### PR DESCRIPTION
**Pull request by guidota**
# Description 

Adds wearable validation to check representations have references to valid content files.

### CODE REVIEW

**Review by agusaldasoro — APPROVED** [review]
Looks good to me! Please check this with someone from dApss team

File: src/validations/wearable.ts [hunk 1]
```diff
@@ -5,6 +5,27 @@ import { ADR_45_TIMESTAMP, calculateDeploymentSize, validateInRow } from '.'
...
 const wearableSizeLimitInMB = 2
+
+/** Validate wearable representations are referencing valid content */
+export const wearableRepresentationContent: Validation = {
+  validate: async ({ deployment }) => {
+    const { entity } = deployment
+    const wearableMetadata = entity.metadata as Wearable
+    const representations = wearableMetadata?.data?.representations
+    if (!representations) return validationFailed('No wearable representations found')
+    if (!entity.content) return validationFailed('No content found')
```
Comment (LautaroPetaccio) [inline src/validations/wearable.ts:16]: There's also another check needed, the representations contain a `mainFile` property that should be included in the `representation.contents`.
https://github.com/decentraland/builder/blob/9492e08439981c71d312daf620b3181fe253c293/src/modules/item/types.ts#L70

Comment (guidota) [reply src/validations/wearable.ts:16]: I believe it is validated in the wearable schema: https://github.com/decentraland/common-schemas/blob/main/src/platform/wearables/representation.ts#L66

Comment (LautaroPetaccio) [reply src/validations/wearable.ts:16]: If that's the case, it seems to be validating everything that we need!

**Pull request merged**